langchain/scripts/create-entrypoints.js

```diff
@@ -34,6 +34,7 @@ const entrypoints = {
   "chains/query_constructor/ir": "chains/query_constructor/ir",
   "chains/sql_db": "chains/sql_db/index",
   "chains/graph_qa/cypher": "chains/graph_qa/cypher",
+  "chains/violation_of_expectations": "chains/violation_of_expectations/violation_of_expectations_chain",
```

Comment:
Filename might need to be `violation_of_expectations/index.ts` or `violation_of_expectations.ts` - I seem to remember some build issues?

Reply:
Also feels like we should put this under `experimental`?